server/game/cards/14.6-Atonement/ShibaPureheart.js

```diff
@@ -0,0 +1,29 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl.js');
+const { CardTypes } = require('../../Constants');
+
+class ShibaPureheart extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            title: 'Honor a character',
+            when: {
+                onConflictDeclared: (event, context) => {
+                    let controller = context.source.controller;
+                    let attacker = event.conflict.attackingPlayer;
+                    if(attacker === controller.opponent) {
+                        return this.game.getConflicts(attacker).filter(conflict => !conflict.passed).length > 1;
+                    }
```

Comment:
Following the cards wording this is technically not correct. Say the Unicorn player would declare his 3rd conflict in the turn, this shouldn't trigger.